docs/apis-tools/operate-api/authentication.md

```diff
@@ -61,7 +61,7 @@ All Operate REST API requests require authentication. To authenticate, generate
 1. [Add an M2M application in Identity](/self-managed/identity/user-guide/additional-features/incorporate-applications.md).
 2. [Add permissions to this application](/self-managed/identity/user-guide/additional-features/incorporate-applications.md) for **Operate API**.
 3. Capture the `Client ID` and `Client Secret` from the application in Identity.
-4. [Generate a token](/self-managed/identity/user-guide/authorizations/generating-m2m-tokens.md) to access the REST API. Provide the `client_id` and `client_secret` from the values you previously captured in Identity.
+4. [Generate a token](/self-managed/identity/user-guide/authorizations/generating-m2m-tokens.md) to access the Operate REST API. Provide the `client_id` and `client_secret` from the values you previously captured in Identity.
```

Comment:
There will be no Operate REST API anymore in 8.8, but just the Camunda 8 REST API.